debian/mariadb-server-10.5.preinst

```diff
@@ -33,14 +33,14 @@ stop_server() {
     if ! pgrep -x mysqld > /dev/null; then return; fi
 
     set +e
-    systemctl stop mysql
+    invoke-rc.d mariadb stop
```

Comment:
`systemctl stop mysql` does not stop multi-instance services and this PR is not changing that. Please don't mix review with feature requests, otherwise this PR will never go in as it would be growing all the time. This PR fixes the fact that `systemctl stop mysql` didn't work in all environments at all, while `invoke-rc.d` works and is the compatible Debian recommended way to do it (and field tested for years in downstream MariaDB packaging)-

Reply:
ack. just remembering other existing bugs while looking through this.